### DISCUSSION TITLE
How to use the library with an email integration

### DISCUSSION THREAD

**Discussion by peoray** [Dave136/vue-email#201] [Q&A]
Not sure what happened but in the past, the docs had a section on how to use it with various email integrations such as resend, etc., but that's not the case anymore.

What do you think is the best way forward based on this?

**Comment by Flowko** [comment]
it makes no sense to keep those docs, as it requires updating and keeping an eye on the actual email sdks, the library gives you the ability to get the `html` and `text` strings, its up to you to follow the email sdk docs and use them where you feel like it,

**Reply by peoray** [reply]
by html and text, you mean the conversions in the usage docs right?

**Reply by Flowko** [reply]
yes 
https://vuemail.net/getting-started/usage#_2-convert-to-html

**Reply by peoray** [reply]
alright gotcha, I was gonna suggest adding the integrations myself but your reasoning makes sense
